lib/restclient/payload.rb

```diff
@@ -116,9 +116,12 @@ def inspect
       end
 
       def short_inspect
-        (size > 500 ? "#{size} byte(s) length" : inspect)
+        if RestClient.limit_payload_log? && size > 500
```

Comment:
Maybe this 500 shouldn't be hardcoded.  Perhaps a CONSTANT?  If needed, we can have a setter to set this value.

Is this 'size' the # of characters or the bytesize of the string?